### ISSUE TITLE
differences with web editor

### ISSUE THREAD

**Issue by jackjackk** [emacs-grammarly/lsp-grammarly#40]
I am trying this package for the first time, so I might be missing something basic. I am comparing the web editor version with the emacs lsp-grammarly version. Below is a visual comparison (emacs left, web editor right). The web editor includes formality checks (you, there's) and inconsistency checks (email vs. e-mail), which don't appear in emacs. Am I misconfiguring anything (I do have `lsp-grammarly-suggestions-informal-pronouns-academic` set)? Am I missing out on other features wrt the web editor? Thanks a lot!

![image](https://user-images.githubusercontent.com/3717222/181564840-f5016e01-b9f0-4c27-b3db-9d93a0092f6d.png)

**Comment by jcs090218** [comment]
I not 100% sure the reason why, but my first guess is your are not using the paid Grammarly Account? Try login, so the language server can be notified by the connected account.

How to login? see https://github.com/emacs-grammarly/lsp-grammarly#-using-a-paid-grammarly-account.

**Comment by jackjackk** [comment]
Thanks, @jcs090218, I did the procedure one via vscode but indeed had a doubt whether it worked or not. Is there a way I can check and be sure that I am logged in and that the premium is correctly working?

**Comment by jcs090218** [comment]
Sorry, I was wrong about the "login" part. Right now, the new the [grammarly-language-server](https://github.com/emacs-grammarly/grammarly-language-server) no longer shares the same credential. The entire login is still work in progress. Sorry for the incorrect instruction and the misleading documentation!

I have tried to input the text to VSCode and I couldn't produce the same result with the web editor as well. So I assumed the issue is from the language server, and not `lsp-grammarly`. Can you try this the same? Let me know if I was wrong about this!

The state of the project is in public beta ([grammarly/grammarly-for-developers](https://github.com/grammarly/grammarly-for-developers)), and entire development is still pretty new and slow. Or maybe the issue is from the parser (tree-sitter-markdown/html).

In conclusion, this seems like an upstream (language-server) bug if we are able to reproduce this issue with VScode.

**Comment by jackjackk** [comment]
You're right: I got the same output as emacs with VSCode. I noticed that by setting the document domain (`lsp-grammarly-domain`) to "academic" (which is a setting I forgot I had in my web profile) I get the formality checks right (you're, there's). I'm still not able to get the consistency checks, which might be an upstream thing at this point. Thanks for the inputs!

**Comment by benthamite** [comment]
I don't want to open a new issue for this, but I'm experiencing a related issue and would be useful to at least be able to determine whether `lsp-grammarly` is using the free or the paid account. @jcs090218, is there a way to check for this?

**Comment by jcs090218** [comment]
I was a bit confused about the login regarding this package, but I think I am wrong about the VSCode login. Here are a few factors that confuses me:

1. The [old grammarly language server](https://github.com/emacs-grammarly/unofficial-grammarly-language-server) shares the same credential. But I don't know if they still work the same way... (I've checked the language server's code, and I don't think it shares the same session)
2. At first, while migrating to the newer server (current) I don't see any differences between the paid and the free version. 
3. VSCode's Grammarly plugins does login but that's all. Since you can't see any differences from the functionality (popup document, code actions, etc), it's hard to tell if the user are using the paid version or not

I've been following the Grammarly language server's activities, but I didn't see any significant changes after migrating this package to the newer language server. 😕 And the author `@znck` seems to be very busy with the [grammarly-for-developers](https://github.com/grammarly/grammarly-for-developers) development (at that it was in BETA) and has not fix some significant issues, e.g., https://github.com/znck/grammarly/issues/302, etc.

I've tried in VSCode and encountered the login bug, so I can't tell if the functionalities have changed. 😕 I'll update you once I notice the differences. 😓
